setup.py

```diff
@@ -55,33 +55,43 @@ def long_description():
     return buf.getvalue()
 
 
+tests_require = (
+    'pytest>=6.2.5',
+    'pytest-network>=0.0.1',
+)
+
 setup(
     author='Ross McFarland',
     author_email='rwmcfa1@gmail.com',
     description=octodns.__doc__,
     entry_points={
         'console_scripts': console_scripts,
     },
-    install_requires=[
+    extras_require={
+        'dev': tests_require + (
+            'build>=0.7.0',
+            'pycodestyle>=2.6.0',
+            'pyflakes>=2.2.0',
+            'readme_renderer[md]>=26.0',
+            'twine>=3.4.2',
+        ),
+    },
+    install_requires=(
         'PyYaml>=4.2b1',
         'dnspython>=1.15.0',
         'fqdn>=1.5.0',
         'natsort>=5.5.0',
         'pycountry>=19.8.18',
         'pycountry-convert>=0.7.2',
```

Comment:
From what I can tell, these are not runtime requirements right? They're only used in `script/generate-geo-data` which I think is not used within an installed package?

Reply:
Will double check they're not needed by anything else and move to the dev side of things if not.

Reply:
Wow. good call on that one. pycountry-convert has a dumb requirements setup that's including a lot of crap that are really test-time requirements etc. https://gist.github.com/ross/61121ab47414fc2334fa1ed480fc3e93 (the pypi linked repo is no more/has moved so gistified it)

Moving it brings requirements.txt down to 7 items https://github.com/octodns/octodns/pull/866/commits/49535fa1045ae18f141d52109998706b690a4a0f